terraform/cross-device/version.tf

```diff
@@ -19,5 +19,9 @@ terraform {
       source  = "hashicorp/google"
       version = ">= 4.69.1, < 6.0.0"
     }
+    random = {
```

Comment:
You don't need this, see my other comment above.

Reply:
Done